packages/@okta/vuepress-site/reference/okta_expression_language/index.md

```diff
@@ -323,454 +323,454 @@ Sample user data:
+| -03:00 | America/Bahia                  |                                                     |
+| -03:00 | America/Belem                  |                                                     |
+| -03:00 | America/Cayenne                |                                                     |
+| -03:00 | America/Fortaleza              |                                                     |
+| -03:00 | America/Godthab                |                                                     |
+| -03:00 | America/Maceio                 |                                                     |
+| -03:00 | America/Miquelon               |                                                     |
+| -03:00 | America/Montevideo             |                                                     |
+| -03:00 | America/Paramaribo             |                                                     |
+| -03:00 | America/Recife                 |                                                     |
+| -03:00 | America/Santarem               |                                                     |
+| -03:00 | America/Sao_Paulo              | Brazil/East                                         |
+| -03:00 | Antarctica/Rothera             |                                                     |
+| -03:00 | Etc/GMT+3                      |                                                     |
+| -02:00 | America/Noronha                | Brazil/DeNoronha                                    |
+| -02:00 | Atlantic/South_Georgia         |                                                     |
+| -02:00 | Etc/GMT+2                      |                                                     |
+| -01:00 | America/Scoresbysund           |                                                     |
+| -01:00 | Atlantic/Azores                |                                                     |
+| -01:00 | Atlantic/Cape_Verde            |                                                     |
+| -01:00 | Etc/GMT+1                      |                                                     |
+| +00:00 | Africa/Abidjan                 |                                                     |
+| +00:00 | Africa/Accra                   |                                                     |
+| +00:00 | Africa/Bamako                  | Africa/Timbuktu                                     |
+| +00:00 | Africa/Banjul                  |                                                     |
+| +00:00 | Africa/Bissau                  |                                                     |
+| +00:00 | Africa/Casablanca              |                                                     |
+| +00:00 | Africa/Conakry                 |                                                     |
+| +00:00 | Africa/Dakar                   |                                                     |
+| +00:00 | Africa/El_Aaiun      |
```

Comment:
Why no third column here and below?

Reply:
Just missed it :(  Lots of tables updated